src/helix/client/client_ext.rs

```diff
@@ -115,6 +115,80 @@ impl<'client, C: crate::HttpClient + Sync + 'client> HelixClient<'client, C> {
         .map(|response| response.data)
     }
 
+    /// Get multiple [Stream](helix::streams::Stream)s from user ids.
+    ///
+    /// # Examples
+    ///
+    /// ```rust, no_run
+    /// # #[tokio::main]
+    /// # async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    /// # let client: helix::HelixClient<'static, twitch_api::client::DummyHttpClient> = helix::HelixClient::default();
+    /// # let token = twitch_oauth2::AccessToken::new("validtoken".to_string());
+    /// # let token = twitch_oauth2::UserToken::from_existing(&client, token, None, None).await?;
+    /// use twitch_api::{types, helix};
+    /// use futures::TryStreamExt;
+    ///
+    /// let channels: &[&types::UserIdRef] = &["123456".into(), "987654".into()];
+    /// let live: Vec<helix::streams::Stream> = client.get_streams_from_ids(&channels, &token).try_collect().await?;
+    ///
+    /// # Ok(()) }
+    /// ```
+    pub fn get_streams_from_ids<T>(
+        &'client self,
+        ids: &'client [&'client types::UserIdRef],
+        token: &'client T,
+    ) -> impl futures::Stream<Item = Result<helix::streams::Stream, ClientError<C>>>
```

Comment:
I hate it

I want the lifetime to be more explicit, should figure out what the correct bound is (if this is not maximally permissable, which I suspect it isn't)